### PR TITLE
feat(healthchecks): add deis healthchecks e2e tests

### LINKED ISSUE #245: implement httpGet liveness check tests
Every example app has a unique port that it listens to inside the container, therefore we cannot create a liveness httpGet healthcheck for https://github.com/deis/workflow-e2e/pull/242. We should come up with a way to either discover the port or standardize the example apps to all listen on a specific port.

### PR DESCRIPTION
**Pull request by kmala**
closes #245

### CODE REVIEW

File: tests/healthcheck_test.go [hunk 1]
```diff
@@ -0,0 +1,205 @@
...
+	"github.com/deis/workflow-e2e/tests/settings"
+
+	. "github.com/onsi/ginkgo"
+	. "github.com/onsi/ginkgo/extensions/table"
+	. "github.com/onsi/gomega"
+	. "github.com/onsi/gomega/gbytes"
+	. "github.com/onsi/gomega/gexec"
+)
+
+var _ = Describe("deis config:healthchecks", func() {
```
Comment (bacongobbler) [inline tests/healthcheck_test.go]: can you change this to just "deis healthchecks"?

Comment (kmala) [inline tests/healthcheck_test.go]: ok

**Comment by bacongobbler** [comment]
also needs a commit squash before merge since there's a merge commit in here

**Comment by kmala** [comment]
@bacongobbler can you check now again?

File: tests/healthcheck_test.go [hunk 2]
```diff
@@ -0,0 +1,205 @@
...
+
+				sess, err = cmd.Start("deis healthchecks:list -a %s", &user, app.Name)
+				Eventually(sess).Should(Say("=== %s Healthchecks", app.Name))
+				Eventually(sess).Should(Say(`Exec Probe\: Command=\[/bin/true]`))
+				Expect(err).NotTo(HaveOccurred())
+				Eventually(sess).Should(Exit(0))
+			})
+
+			// disable httpGet liveness checks as it won't pass for every app
+			// TODO(bacongobbler): somehow determine *what* port we need to set the liveness check for this app
```
Comment (vdice) [inline tests/healthcheck_test.go]: I believe @bacongobbler created the following ticket for this `TODO` -- can we reference the issue number here? https://github.com/deis/workflow-e2e/issues/245

File: tests/healthcheck_test.go [hunk 3]
```diff
@@ -0,0 +1,205 @@
...
+			})
+		})
+	})
+
+	DescribeTable("any user can get command-line help for healthchecks", func(command string, expected string) {
+		sess, err := cmd.Start(command, nil)
+		Eventually(sess).Should(Say(expected))
+		Expect(err).NotTo(HaveOccurred())
+		Eventually(sess).Should(Exit(0))
+		// TODO: test that help output was more than five lines long
```
Comment (vdice) [inline tests/healthcheck_test.go]: can we reference the correlating GH ticket (https://github.com/deis/workflow-e2e/issues/244) here?

**Comment by vdice** [comment]
both comments purely around `TODO` comments, otherwise LGTM.

**Comment by krancour** [comment]
Commits still ned squashing here.

**Pull request merged**